java/src/main/java/com/powsybl/dataframe/network/adders/ShuntDataframeAdder.java

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.dataframe.network.adders;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.dataframe.SeriesMetadata;
+import com.powsybl.dataframe.update.UpdatingDataframe;
+import com.powsybl.iidm.network.*;
+
+import java.util.List;
+
+/**
+ * @author Yichen TANG <yichen.tang at rte-france.com>
+ * @author Etienne Lesot <etienne.lesot at rte-france.com>
+ * @author Sylvain Leclerc <sylvain.leclerc@rte-france.com>
+ */
+public class ShuntDataframeAdder implements NetworkElementAdder {
+
```

Comment:
In general, for shunt, it could be great to have a way to only set a B from a dataframe, that will create a shunt understandable by Powsybl.

Reply:
See discussion synthesis about default values in comment below.